src/promptflow/tests/sdk_cli_test/e2etests/test_flow_serve.py

```diff
@@ -115,6 +119,7 @@ def test_chat_swagger(serving_client_llm_chat):
     }
 
 
+@pytest.mark.skipif(RecordStorage.is_replaying_mode(), reason="Doesn't support serve in replay mode")
```

Comment:
maybe add a class to group all these tests?

Reply:
+1 on this, or open a task tracking this change; I think we should always use class for testing.

Reply:
Create Task for skipped tests. cannot easy fix